[attr] Build failure

Package: attr:x64-linux -> 2.5.1

**Host Environment**

- Host: x64-linux
- Compiler: GNU 12.2.0
-    vcpkg-tool version: 2023-07-19-814b7ec837b59f1c8778f72351c1dd7605983cd2
    vcpkg-scripts version: 05721705 2023-08-02 (28 minutes ago)

**To Reproduce**

`vcpkg install iceoryx`

**Failure logs**

```
-- Using cached attr-2.5.1.tar.xz.
-- Cleaning sources at /home/abdul/vcpkg/buildtrees/attr/src/attr-2-188a7b0fb4.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/abdul/vcpkg/downloads/attr-2.5.1.tar.xz
-- Using source at /home/abdul/vcpkg/buildtrees/attr/src/attr-2-188a7b0fb4.clean
-- Found external ninja('1.11.1').
-- Getting CMake variables for x64-linux-dbg
-- Getting CMake variables for x64-linux-rel
-- Generating configure for x64-linux
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /usr/bin/autoreconf -vfi
    Working Directory: /home/abdul/vcpkg/buildtrees/attr/src/attr-2-188a7b0fb4.clean/
    Error code: 1
    See logs for more information:
      /home/abdul/vcpkg/buildtrees/attr/autoconf-x64-linux-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_make.cmake:635 (vcpkg_execute_required_process)
  ports/attr/portfile.cmake:14 (vcpkg_configure_make)
  scripts/ports.cmake:147 (include)



```
<details><summary>/home/abdul/vcpkg/buildtrees/attr/autoconf-x64-linux-err.log</summary>

```
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: running: autopoint --force
Can't exec "autopoint": No such file or directory at /usr/share/autoconf/Autom4te/FileUtils.pm line 293.
autoreconf: error: autopoint failed with exit status: 1
```
</details>

**Additional context**

--- a/ports/attr/portfile.cmake
+++ b/ports/attr/portfile.cmake
@@ -1,5 +1,3 @@
-vcpkg_minimum_required(VERSION 2022-10-12) # for ${VERSION}
-
 vcpkg_download_distfile(ARCHIVE
     URLS "http://download.savannah.nongnu.org/releases/attr/attr-${VERSION}.tar.xz"
     FILENAME "attr-${VERSION}.tar.xz"
@@ -8,12 +6,23 @@ vcpkg_download_distfile(ARCHIVE
 
 vcpkg_extract_source_archive(
     SOURCE_PATH
-    ARCHIVE ${ARCHIVE}
+    ARCHIVE "${ARCHIVE}"
 )
+
+vcpkg_list(SET options)
+if("nls" IN_LIST FEATURES)
+    vcpkg_list(APPEND options "--enable-nls")
+    vcpkg_add_to_path(PREPEND "${CURRENT_HOST_INSTALLED_DIR}/tools/gettext/bin")
+else()
+    set(ENV{AUTOPOINT} true) # true, the program
+    vcpkg_list(APPEND options "--disable-nls")
+endif()
 
 vcpkg_configure_make(
     SOURCE_PATH "${SOURCE_PATH}"
     AUTOCONFIG
+    OPTIONS
+        ${options}
 )
 
 vcpkg_install_make()

--- a/ports/attr/vcpkg.json
+++ b/ports/attr/vcpkg.json
@@ -1,8 +1,24 @@
 {
   "name": "attr",
   "version-semver": "2.5.1",
+  "port-version": 1,
   "description": "Commands for Manipulating Filesystem Extended Attributes",
   "homepage": "http://savannah.nongnu.org/projects/attr",
   "license": "LGPL-2.1-or-later",
-  "supports": "linux"
+  "supports": "linux",
+  "features": {
+    "nls": {
+      "description": "Enable native language support",
+      "dependencies": [
+        "gettext",
+        {
+          "name": "gettext",
+          "host": true,
+          "features": [
+            "tools"
+          ]
+        }
+      ]
+    }
+  }
 }

--- a/versions/a-/attr.json
+++ b/versions/a-/attr.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "2348d8cea7b1c70c26139bbf366d57830cea8bf5",
+      "version-semver": "2.5.1",
+      "port-version": 1
+    },
+    {
       "git-tree": "99ae42f2118c5c6d128b332dd5e914aeba6702d9",
       "version-semver": "2.5.1",
       "port-version": 0

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -298,7 +298,7 @@
     },
     "attr": {
       "baseline": "2.5.1",
-      "port-version": 0
+      "port-version": 1
     },
     "aubio": {
       "baseline": "2022-01-26",

